agent/acs/handler/attach_eni_handler.go

```diff
@@ -107,6 +109,13 @@ func (attachENIHandler *attachENIHandler) handleSingleMessage(message *ecsacs.At
 	if err != nil {
 		seelog.Warnf("Failed to ack request with messageId: %s, error: %v", aws.StringValue(message.MessageId), err)
 	}
+
+	// Check if this is a duplicate message
+	if _, ok := attachENIHandler.state.ENIByMac(aws.StringValue(message.ElasticNetworkInterfaces[0].MacAddress)); ok {
+		seelog.Info("Duplicate ENIAttachment message, ENIAttachment is already managed by agent, mac: %s", aws.StringValue(message.ElasticNetworkInterfaces[0].MacAddress))
```

Comment:
needs to be `Infof`. Also, can you make this more user friendly? `"Duplicate ENI attachment message for ENI with MAC address: %s"`